handler/create_user.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	db, err := sqlx.Connect("postgres", os.Getenv("POSTGRES_DSN"))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte(fmt.Sprintf("db connection error: %s", err.Error())))
+		return
+	}
+
+	err = r.ParseForm()
```

Comment:
are you sure the request will be a form? iirc, [swr](swr.vercel.app) prefers json